examples/xmlrpc/xmlrpc_main.c

```diff
@@ -133,7 +138,8 @@ static char *xmlrpc_findbody(char *buf)
  *
  ****************************************************************************/
 
-static int xmlrpc_getheader(char *buffer, char *header, char *value, int size)
+static int xmlrpc_getheader(char *buffer, char *header,
+                            char *value, int size)
```

Comment:
Not related to this PR, but will leave here for history
```suggestion
static int xmlrpc_getheader(FAR char *buffer, FAR char *header,
                            FAR char *value, int size)
```